assets/keymaps/linux/jetbrains.json

```diff
@@ -91,7 +91,7 @@
   {
     "context": "Workspace",
     "bindings": {
-      "ctrl-shift-f12": "workspace::CloseAllDocks",
```

Comment:
Let's use that everywhere else too, this seems quite convenient.

Reply:
All other keymaps, yeah.

Good point about the menu: let's swap things there too?
Feels that the "close" action is fine to remove from the UI/bindings as the toggle works better and allows to achieve the same.
Removing this entirely is not that good, as we might break someone's custom bindings.